fabric-networking-api-v1/src/client/java/net/fabricmc/fabric/impl/networking/client/ClientLoginNetworkAddon.java

```diff
@@ -103,6 +103,11 @@ protected void handleRegistration(Identifier channelName) {
 	protected void handleUnregistration(Identifier channelName) {
 	}
 
+	@Override
+	public @Nullable String getBrand() {
```

Comment:
Ideally this wouldnt exist on the login addon, could the root method go in `ClientCommonNetworkAddon`? Im not too worried about this as its impl, and can easily be refactored later. Or even sooner before this PR goes in as its own thing.

Reply:
Originally I wanted to use it purely for changing the message (through ended up using the same one for everyone), so yeah I can drop this for now